src/_pytest/helpconfig.py

```diff
@@ -191,9 +194,13 @@ def showhelp(config: Config) -> None:
             tw.write(" " * (indent_len - spec_len - 2))
             wrapped = textwrap.wrap(help, columns - indent_len, break_on_hyphens=False)
 
-            tw.line(wrapped[0])
-            for line in wrapped[1:]:
-                tw.line(indent + line)
+            # If a help string wasn't specified, just write an empty line.
```

Comment:
This would definitely solve it, but do we want to allow this? From an end user perspective I feel like an empty line would confuse me, and I would think that the feature is unimplemented, or that pytest is broken in general and not outputting information.

Reply:
Do you think it'd be better to throw an error on an empty help or perhaps display a message in its place indicating it was empty? Open to all suggestions as I agree it may come across as a bug/something has broken.

Reply:
I’m personally leaning towards throwing an error clearly indicating to the user that the help text cannot be empty. I feel like we should never assume for the end user. Also, I’m unsure as to what an appropriate default help message would even contain

Reply:
I was thinking something like `[No description provided]` so the end user knows the state of the option. But, happy to change to throwing an error (a `TypeError` like a missing function param?) on help being `None` or `""`, let me know what you would like.